listing.php

```diff
@@ -206,12 +205,14 @@ function showTreeDir($svnrep, $path, $rev, $peg, $listing) {
 	}
 
 	$vars['compare_box'] = ''; // Set blank once in case tree view is not enabled.
-	return showDirFiles($svnrep, $subs, 0, $limit, $rev, $peg, $listing, 0, $config->treeView);
+	//return showDirFiles($svnrep, $subs, 0, $limit, $rev, $peg, $listing, 0, $config->treeView);
```

Comment:
No need for such comments, we have source control.

Reply:
Removed as these were the changes for #104